rasa_nlu/extractors/crf_entity_extractor.py

```diff
@@ -375,17 +428,23 @@ def _from_json_to_crf(self,
                           ):
         # type: (...) -> List[Tuple[Text, Text, Text, Text]]
         """Convert json examples to format of underlying crfsuite."""
-        from spacy.gold import GoldParse
 
-        doc = message.get("spacy_doc")
-        gold = GoldParse(doc, entities=entity_offsets)
-        ents = [l[5] for l in gold.orig_annot]
+        if self.pos_features:
+            from spacy.gold import GoldParse
+
+            doc = message.get("spacy_doc")
+            gold = GoldParse(doc, entities=entity_offsets)
+            ents = [l[5] for l in gold.orig_annot]
+        else:
+            tokens = message.get("tokens")
+            ents = self._bilou_tags_from_offsets(tokens, entity_offsets)
```

Comment:
is there a reason not to use our bilou always?

Reply:
we need GoldParse in case the tokens come from spacy_doc. The spacy tokens have different attributes from the Rasa tokens. 
See https://github.com/explosion/spaCy/blob/master/spacy/gold.pyx#L520 and 
https://github.com/RasaHQ/rasa_nlu/blob/6c5149f0ecdb6ec982ee0214177711d3d4154b34/rasa_nlu/extractors/crf_entity_extractor.py#L459